content/e/ppw/2024-07-13-ppw-ale-grzeje.md

```diff
@@ -70,12 +70,12 @@ Prior to the show, PpW held a 5-day open seminar with [Jacob Crane](@/w/jacob-cr
 ### References
 
 * [Cagematch event page](https://www.cagematch.net/?id=1&nr=397548)
-* [YouTube livestream of the event](https://www.youtube.com/watch?v=UEQkSKMekCs&ab_channel=PpWEwenementWrestling)
+* [Full show on YouTube](https://www.youtube.com/watch?v=UEQkSKMekCs&t=14480s) (at the 4:01:18 mark Michael announces the next show)
 * [Facebook announcement post](https://www.facebook.com/photo/?fbid=976634991133560&set=a.499910772139320)
 * [Facebook event: Summer Course](https://www.facebook.com/events/1409438239775557/?acontext=%7B%22event_action_history%22%3A%5B%5D%7D)
 * [Facebook announcement of Crane vs Queen](https://www.facebook.com/OficjalnePPW/posts/pfbid0ihqTqNJFNkqXJR6xia7jrVyX78HtjWan8dvRPDa3s3aNb5xV79TKZusRbVger6bGl)
 * [Facebook announcement of Goblin vs Legia](https://www.facebook.com/OficjalnePPW/posts/pfbid025CV6NVnhWE3BiijAMrECRcY6K68u2FPRhoo48M45ZH5jFpUWf2xjuoEz25nvYZrgl)
-* [KULTURA FIZYCZNA VOL.2](https://komuna.warszawa.pl/events/kultura-fizyczna-vol-2/), Teatr Komuna's second event celebrating sport and physicality, including PpW Ale Grzeje.
+* [Kultura Fizyczna vol.2](https://komuna.warszawa.pl/events/kultura-fizyczna-vol-2/), Teatr Komuna's second event celebrating sport and physicality, including PpW Ale Grzeje.
```

Comment:
Komuna's original title, wanted to preserve the all-caps. However, this can no longer be found on their page, after less than a year (nor on IA) - a lesson in why should we put important things into an archive.